<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>WeatherWidget</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/sun.jpg</normaloff>../../Downloads/sun.jpg</iconset>
  </property>
  <widget class="QFrame" name="fr_forecast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>forecast</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QFrame" name="fr_forecast_day_0">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>121</width>
      <height>149</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>forecast_today</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_fr_date_0">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>date</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_fr_temp_max_0">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>55</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>max: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_fr_temp_min_0">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>30</y>
       <width>55</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>min: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_forecast_day_1">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>40</y>
      <width>121</width>
      <height>149</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>forecast_tomorrow</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_fr_date_1">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>date</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_fr_temp_max_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>55</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>max: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_fr_temp_min_1">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>30</y>
       <width>55</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>min: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="fr_forecast_day_2">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>40</y>
      <width>121</width>
      <height>149</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>forecast_day_after_tomorrow</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_fr_date_3">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>date</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_fr_temp_max_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>55</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>max: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_fr_temp_min_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>30</y>
       <width>55</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>min: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="fr_current">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>current_weather</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_cur_icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>136</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>icon</string>
    </property>
    <property name="text">
     <string>icon_default</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>30</y>
      <width>136</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lb_cur_temp_c">
       <property name="toolTip">
        <string>temperature_celcius</string>
       </property>
       <property name="text">
        <string>temp_c:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="fr_extra">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>extra_info</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_city">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>84</width>
      <height>38</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>city_name</string>
    </property>
    <property name="text">
     <string>city_default</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_fr">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>115</y>
     <width>205</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Forecast</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_cw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>269</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Current Weather</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
